src/ngrx/7-create-api-effects/create-api-effects.md

```diff
@@ -34,29 +36,50 @@ TODO
 
 </details>
 
-## P1: Solution
+## P1: What You Need to Know
 
-<details>
-<summary>src/app/store/login/login.effects.ts</summary>
+[NgRx Effects](https://ngrx.io/guide/effects) is a side-effect model that utilizes RxJS to react to Actions being dispatched to manage side-effects such as network requests, web socket messages and time-based events. One thing that Effects are not responsible for is updating state; this is a responsiblity for Reducers ([learn-ngrx/create-reducer]).
 
-@diff ./login.effects-error-message-helper.ts ./login.effects-login-effect.ts only
+> Note that, for a given Action, Effects will always happen after the state has been updated by the Reducer.
```

Comment:
I think the first comma is superfluous here

Reply:
I had to google "superfluous" and yea I agree. Good catch :P